plugins/Minecraft/index.js

```diff
@@ -0,0 +1,86 @@
+const fetch = require('node-fetch');
+
+export default function (bastion, opt = {}) {
+    // const cmd = bastion.command("minecraft")
+    // const help = `${cmd} (description)`
+    return [{
+            command: "mc_info",
+            helpOnEmpty: false,
+            // restrict: [bastion.channels.gaming],
+            restrictMessage: "",
+            options: "",
+            validate(context, msg) {
+                if (msg.includes(" ")) return 'Please just do the command'
+            },
+
+            async resolve(context, msg) {
+                const mcURL = "https://mcapi.us/server/status?ip=51.161.122.136&port=25595";
+
+                let response = await fetch(mcURL);
+                if (response.ok) {
+                    let jsonResponse = await response.json();
+                    if (jsonResponse.online === false) {
+                        return "The server is either offline or the bot is broken. Please dm @Devrim#1557 if the server is online."
+                    } else {
```

Comment:
No need to `} else {` here since the previous statement returns. This can prevent deep nesting

```js
if (!jsonResponseOnline) {
   return "The server is either offline or the bot is broken. Please dm @Devrim#1557 if the server is online."
}

let serverStatus = // continue
```